reactDashboard/src/components/DataDefinitions/DataDefinitions.jsx

```diff
@@ -77,48 +81,50 @@ function DataDefinitions() {
           <Card.Body className="data-definitions-body">
             <p className="definitions-heading">Reporting and Specimen Dates</p>
             <hr />
-            <ul>
-              <li>
+              <p>
                 <span className="defined-term">Reported Dates</span>: Since the
                 time taken to test samples and report the results varies, new
                 cases reported on a daily basis in the headline summary figures
                 on the dashboard pages may be distributed across a range of
                 specimen dates.
-              </li>
-              <li>
+              </p>
+              <br/>
+              <p>
```

Comment:
the span is no longer needed - just move the class name up to the p

Reply:
that is what makes the defined term bold - moving the classname would make the whole paragraph bold?

Reply:
Ah, yeah - my bad

Reply:
If it's just making the text bold, why not use strong?

Reply:
will change it